### PR TITLE
Beta CTF

### PR DESCRIPTION
**Pull request by AvrooVulcan**
Added regex for beta ctf game

### CODE REVIEW

**Comment by AvrooVulcan** [comment]
Could possibly add "CTF" into the regex, but might be a bit generic

File: cdbot/cogs/cyber.py [hunk 1]
```diff
@@ -133,7 +136,7 @@ def __init__(self, bot: Bot):
             level_num = int(base)
             base = "Headquarters"
         if level_num == 13 and challenge_num == 1:
-            content = "13.1 is a No Flag Zone™ 🙅⛔⚔️"
+            content = "13.1 is a No Flag Zone™ ðﾟﾙﾅ⛔⚔️"
```
Comment (picapi) [inline cdbot/cogs/cyber.py]: As a note; the formatting of an emoji here appears to have been changed / become invalid.

File: cdbot/cogs/cyber.py [hunk 2]
```diff
@@ -51,6 +51,9 @@ class Cyber(Cog):
          "We’re currently allocating students to their preferred locations so it’s an ongoing process! "
          "We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
          ),
+
+        # Beta date
+        (r"^.*\bbeta\b.*(end|finish|close)\b.*$", "Cyberstart Cloud CTF ends on the May 13th 2019 at 12:00pm."),
```
Comment (joker314) [inline cdbot/cogs/cyber.py]:
```suggestion
        (r"^.*\bbeta\b.*(end|finish|close)\b.*$", "Cyberstart Cloud CTF ends on the 13th May 2019 at 12:00pm."),
```

**Comment by AvrooVulcan** [comment]
@joker314 committed, cheers

**Pull request merged**